starfish/imagestack/imagestack.py

```diff
@@ -335,19 +335,19 @@ def sel(self, indexers: Mapping[Indices, Union[int]]):
         Examples
         --------
 
-        Create an Imagestack using the ``synthetic_stack`` method::
```

Comment:
Looks like we have a mix of ending these Example introductions with `::`, :`:, and no colon as here.

Reply:
Can someone explain what the differences in all these might be?  Hard to google this sort of thing. :(

Reply:
Yeah, I don't know. Yours certainly looked ok. But `::` did as well. I didn't look up what `:` came out looking like. Probably just best if we pick one we like and all agree on it.